data/campaigns/tutorial02_warfare.wmf/scripting/texts.lua

```diff
@@ -126,6 +126,37 @@ trainingcamp2 = {
    )
 }
 
+scouting1 = {
+   position = "topright",
+   title = _"Scout's Hut",
+   body = (
+      h1(_"Explore Your Surroundings") ..
+      p(_[[Large part of the map is not explored yet. It might be a good idea to know which direction is best for expanding our empire.]]) ..
```

Comment:
```suggestion
      p(_[[Large parts of the map haven’t been explored yet. It might be a good idea to know which direction is best for expanding our empire.]]) ..
```